### PR TITLE
Translate Eng docs to Chinese.

### PR DESCRIPTION
**Pull request by marcuswang6**
### Changelog category (leave one):

- Documentation (changelog entry is not required)

### Documentation entry for user-facing changes

- [x] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by robot-ch-test-poll1** [comment]
*This is an automated comment for commit ef044842c5079a3d1a6adea3289fe9d64e594b73 with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/58892/ef044842c5079a3d1a6adea3289fe9d64e594b73/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/58892/ef044842c5079a3d1a6adea3289fe9d64e594b73/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

File: docs/zh/faq/general/ne-tormozit.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+  \u201D 是什么意思？"
 toc_hidden: true
 sidebar_position: 11
 ---
 
-# What Does “Не тормозит” Mean? {#what-does-ne-tormozit-mean}
+# “Не тормозит” 是什么意思？ {#what-does-ne-tormozit-mean}
 
-This question usually arises when people see official ClickHouse t-shirts. They have large words **“ClickHouse не тормозит”** on the front.
+这个问题通常出现在人们看到官方 ClickHouse T恤时。它们的正面印有大字**“ClickHouse Не тормозит”**。
```
Comment (hanfei1991) [inline docs/zh/faq/general/ne-tormozit.md:11]:
```suggestion
这个问题通常出现在人们看到官方 ClickHouse T恤时。它们的正面印有大字**“ClickHouse не тормозит”**。
```

File: docs/zh/faq/general/ne-tormozit.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
 
-Before ClickHouse became open-source, it has been developed as an in-house storage system by the largest Russian IT company, [Yandex](https://yandex.com/company/). That’s why it initially got its slogan in Russian, which is “не тормозит” (pronounced as “ne tormozit”). After the open-source release we first produced some of those t-shirts for events in Russia and it was a no-brainer to use the slogan as-is.
+在 ClickHouse 开源之前，它作为俄罗斯最大的 IT 公司 [Yandex](https://yandex.com/company/) 的内部存储系统而开发。这就是为什么它最初获得了俄文口号“Не тормозит”（发音为“ne tormozit”）。在开源发布后，我们首先为俄罗斯的活动制作了一些这样的 T恤，使用原汁原味的口号是理所当然的。
```
Comment (hanfei1991) [inline docs/zh/faq/general/ne-tormozit.md:13]:
```suggestion
在 ClickHouse 开源之前，它作为俄罗斯最大的 IT 公司 [Yandex](https://yandex.com/company/) 的内部存储系统而开发。这就是为什么它最初获得了俄文口号“не тормозит”（发音为“ne tormozit”）。在开源发布后，我们首先为俄罗斯的活动制作了一些这样的T恤，使用原汁原味的口号是理所当然的。
```

File: docs/zh/faq/general/ne-tormozit.md [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
 
-One of the following batches of those t-shirts was supposed to be given away on events outside of Russia and we tried to make the English version of the slogan. Unfortunately, the Russian language is kind of elegant in terms of expressing stuff and there was a restriction of limited space on a t-shirt, so we failed to come up with good enough translation (most options appeared to be either long or inaccurate) and decided to keep the slogan in Russian even on t-shirts produced for international events. It appeared to be a great decision because people all over the world get positively surprised and curious when they see it.
+其中一批这样的 T恤原本打算在俄罗斯之外的活动中赠送，我们尝试制作口号的英文版本。不幸的是，俄语在表达事物方面有种优雅，而且 T恤上的空间有限，所以我们未能提出足够好的翻译（大多数选项要么太长，要么不够准确），并决定即使在为国际活动制作的 T恤上也保留俄文口号。这被证明是一个绝妙的决定，因为全世界的人们看到它时都会感到惊喜和好奇。
```
Comment (hanfei1991) [inline docs/zh/faq/general/ne-tormozit.md:15]:
```suggestion
其中一批这样的T恤原本打算在俄罗斯之外的活动中赠送，我们尝试制作口号的英文版本。不幸的是，俄语在表达方面有些优雅，而且T恤上的空间有限，所以我们未能提出足够好的翻译（大多数选项要么太长，要么不够准确），并决定即使在为国际活动制作的T恤上也保留俄文口号。这被证明是一个绝妙的决定，因为全世界的人们看到它时都会感到惊喜和好奇。
```

File: docs/zh/faq/general/ne-tormozit.md [hunk 4]
```diff
@@ -1,27 +1,27 @@
...
 
-So, what does it mean? Here are some ways to translate *“не тормозит”*:
+那么，它是什么意思呢？以下是翻译“Не тормозит”的一些方式：
```
Comment (hanfei1991) [inline docs/zh/faq/general/ne-tormozit.md:17]:
```suggestion
那么，它是什么意思呢？以下是翻译“не тормозит”的一些方式：
```

File: docs/zh/faq/general/ne-tormozit.md [hunk 5]
```diff
@@ -1,27 +1,27 @@
...
 
--   If you translate it literally, it’d be something like *“ClickHouse does not press the brake pedal”*.
--   If you’d want to express it as close to how it sounds to a Russian person with IT background, it’d be something like *“If your larger system lags, it’s not because it uses ClickHouse”*.
--   Shorter, but not so precise versions could be *“ClickHouse is not slow”*, *“ClickHouse does not lag”* or just *“ClickHouse is fast”*.
+-   如果你直译，那就是“ClickHouse 不拖后腿”。
```
Comment (hanfei1991) [inline docs/zh/faq/general/ne-tormozit.md:19]:
```suggestion
-   如果你直译，那就是“ClickHouse 不踩刹车”。
```

File: docs/zh/faq/integration/json-import.md [hunk 6]
```diff
@@ -1,35 +1,35 @@
...
+## 实用设置 {#useful-settings}
 
--   `input_format_skip_unknown_fields` allows to insert JSON even if there were additional fields not present in table schema (by discarding them).
--   `input_format_import_nested_json` allows to insert nested JSON objects into columns of [Nested](../../sql-reference/data-types/nested-data-structures/nested.md) type.
+-   `input_format_skip_unknown_fields` 允许插入 JSON，即使存在表格架构中未出现的额外字段（通过丢弃它们）。
+-   `input_format_import_nested_json` 允许将嵌套 JSON 对象插入到 [Nested](../../sql-reference/data-types/nested-data-structures/nested.md) 类型的列中。
 
 :::note
-Settings are specified as `GET` parameters for the HTTP interface or as additional command-line arguments prefixed with `--` for the `CLI` interface.
+对于 HTTP 接口，设置作为 GET 参数指定；对于 CLI 接口，则作为前缀为 -- 的附加命令行参数。
```
Comment (hanfei1991) [inline docs/zh/faq/integration/json-import.md:34]:
```suggestion
对于 HTTP 接口，设置作为 `GET` 参数指定；对于 `CLI` 接口，则作为前缀为 -- 的附加命令行参数。
```

File: docs/zh/faq/operations/delete-old-data.md [hunk 7]
```diff
@@ -1,44 +1,44 @@
...
-# Is It Possible to Delete Old Records from a ClickHouse Table? {#is-it-possible-to-delete-old-records-from-a-clickhouse-table}
+# 是否可以从ClickHouse表中删除旧记录？ {#is-it-possible-to-delete-old-records-from-a-clickhouse-table}
 
-The short answer is “yes”. ClickHouse has multiple mechanisms that allow freeing up disk space by removing old data. Each mechanism is aimed for different scenarios.
+简短的答案是“可以”。ClickHouse具有多种机制，允许通过删除旧数据来释放磁盘空间。每种机制都针对不同的场景。
 
 ## TTL {#ttl}
 
-ClickHouse allows to automatically drop values when some condition happens. This condition is configured as an expression based on any columns, usually just static offset for any timestamp column.
+ClickHouse允许在某些条件发生时自动删除值。这个条件被配置为基于任何列的表达式，通常只是针对任何时间戳列的静态偏移量。
```
Comment (hanfei1991) [inline docs/zh/faq/operations/delete-old-data.md:14]:
```suggestion
ClickHouse 允许在某些条件发生时自动删除值。这个条件被配置为基于任何列的表达式，通常只是针对任何时间戳列的静态偏移量。
```

File: docs/zh/faq/operations/delete-old-data.md [hunk 8]
```diff
@@ -1,44 +1,44 @@
...
 
-The key advantage of this approach is that it does not need any external system to trigger, once TTL is configured, data removal happens automatically in background.
+这种方法的主要优势是它不需要任何外部系统来触发，一旦配置了TTL，数据删除就会自动在后台发生。
```
Comment (hanfei1991) [inline docs/zh/faq/operations/delete-old-data.md:16]:
```suggestion
这种方法的主要优势是它不需要任何外部系统来触发，一旦配置了 TTL，数据删除就会自动在后台发生。
```

File: docs/zh/faq/operations/delete-old-data.md [hunk 9]
```diff
@@ -1,44 +1,44 @@
...
 
 :::note
-TTL can also be used to move data not only to [/dev/null](https://en.wikipedia.org/wiki/Null_device), but also between different storage systems, like from SSD to HDD.
+TTL也可以用来将数据移动到非 [/dev/null](https://en.wikipedia.org/wiki/Null_device) 的不同存储系统，例如从SSD到HDD。
```
Comment (hanfei1991) [inline docs/zh/faq/operations/delete-old-data.md:19]:
```suggestion
TTL 也可以用来将数据移动到非 [/dev/null](https://en.wikipedia.org/wiki/Null_device) 的不同存储系统，例如从 SSD 到 HDD。
```

File: docs/zh/faq/operations/delete-old-data.md [hunk 10]
```diff
@@ -1,44 +1,44 @@
...
 :::
 
-More details on [configuring TTL](../../engines/table-engines/mergetree-family/mergetree.md#table_engine-mergetree-ttl).
+有关 [配置TTL](../../engines/table-engines/mergetree-family/mergetree.md#table_engine-mergetree-ttl) 的更多详细信息。
```
Comment (hanfei1991) [inline docs/zh/faq/operations/delete-old-data.md:22]:
```suggestion
有关 [配置 TTL](../../engines/table-engines/mergetree-family/mergetree.md#table_engine-mergetree-ttl) 的更多详细信息。
```

File: docs/zh/faq/operations/delete-old-data.md [hunk 11]
```diff
@@ -1,44 +1,44 @@
...
 
 ## ALTER DELETE {#alter-delete}
 
-ClickHouse does not have real-time point deletes like in [OLTP](https://en.wikipedia.org/wiki/Online_transaction_processing) databases. The closest thing to them are mutations. They are issued as `ALTER ... DELETE` or `ALTER ... UPDATE` queries to distinguish from normal `DELETE` or `UPDATE` as they are asynchronous batch operations, not immediate modifications. The rest of syntax after `ALTER TABLE` prefix is similar.
+ClickHouse没有像[OLTP](https://en.wikipedia.org/wiki/Online_transaction_processing)数据库那样的实时点删除。最接近的东西是突变。它们被发布为`ALTER ... DELETE`或`ALTER ... UPDATE`查询，以区别于普通的`DELETE`或`UPDATE`，因为它们是异步批处理操作，而不是立即修改。`ALTER TABLE`前缀后的其余语法相似。
```
Comment (hanfei1991) [inline docs/zh/faq/operations/delete-old-data.md:26]:
```suggestion
ClickHouse没有像[OLTP](https://en.wikipedia.org/wiki/Online_transaction_processing)数据库那样的实时点删除。最接近的东西是 `Mutation`，执行 `ALTER ... DELETE` 或 `ALTER ... UPDATE` 查询，以区别于普通的`DELETE`或`UPDATE`。因为它们是异步批处理操作，而不是立即修改。`ALTER TABLE`前缀后的其余语法相似。
```

File: docs/zh/faq/operations/delete-old-data.md [hunk 12]
```diff
@@ -1,44 +1,44 @@
...
 
-`ALTER DELETE` can be issued to flexibly remove old data. If you need to do it regularly, the main downside will be the need to have an external system to submit the query. There are also some performance considerations since mutation rewrite complete parts even there’s only a single row to be deleted.
+`ALTER DELETE`可以灵活地用来删除旧数据。如果你需要定期这样做，主要缺点将是需要有一个外部系统来提交查询。还有一些性能方面的考虑，因为即使只有一行要被删除，突变也会重写完整部分。
 
-This is the most common approach to make your system based on ClickHouse [GDPR](https://gdpr-info.eu)-compliant.
+这是使基于ClickHouse的系统符合[GDPR](https://gdpr-info.eu)的最常见方法。
 
-More details on [mutations](../../sql-reference/statements/alter.md/#alter-mutations).
+有关[突变](../../sql-reference/statements/alter.md/#alter-mutations)的更多详细信息。
```
Comment (hanfei1991) [inline docs/zh/faq/operations/delete-old-data.md:32]:
```suggestion
有关[mutations](../../sql-reference/statements/alter.md/#alter-mutations)的更多详细信息。
```

File: docs/zh/faq/operations/delete-old-data.md [hunk 13]
```diff
@@ -1,44 +1,44 @@
...
 
 ## DROP PARTITION {#drop-partition}
 
-`ALTER TABLE ... DROP PARTITION` provides a cost-efficient way to drop a whole partition. It’s not that flexible and needs proper partitioning scheme configured on table creation, but still covers most common cases. Like mutations need to be executed from an external system for regular use.
+`ALTER TABLE ... DROP PARTITION`提供了一种成本效率高的方式来删除整个分区。它不是那么灵活，需要在创建表时配置适当的分区方案，但仍然涵盖了大多数常见情况。像突变一样，需要从外部系统执行以进行常规使用。
```
Comment (hanfei1991) [inline docs/zh/faq/operations/delete-old-data.md:36]:
```suggestion
`ALTER TABLE ... DROP PARTITION`提供了一种成本效率高的方式来删除整个分区。它不是那么灵活，需要在创建表时配置适当的分区方案，但仍然涵盖了大多数常见情况。像 mutations 一样，需要从外部系统执行以进行常规使用。
```

File: docs/zh/faq/operations/delete-old-data.md [hunk 14]
```diff
@@ -1,44 +1,44 @@
...
+有关[操作分区](../../sql-reference/statements/alter/partition.mdx/#alter_drop-partition)的更多详细信息。
 
 ## TRUNCATE {#truncate}
 
-It’s rather radical to drop all data from a table, but in some cases it might be exactly what you need.
+从表中删除所有数据是相当激进的，但在某些情况下可能正是您所需要的。
 
-More details on [table truncation](../../sql-reference/statements/truncate.md).
\ No newline at end of file
+有关[表截断](../../sql-reference/statements/truncate.md)的更多详细信息。
```
Comment (hanfei1991) [inline docs/zh/faq/operations/delete-old-data.md:44]:
```suggestion
有关[truncate](../../sql-reference/statements/truncate.md)的更多详细信息。
```

**Review by hanfei1991 — APPROVED** [review]

**Pull request merged**